src/main/java/com/untamedears/JukeAlert/model/LoggedAction.java

```diff
@@ -59,4 +65,30 @@ public static LoggedAction getFromId(int id) {
             default: return UNKNOWN;
         }
     }
+    
```

Comment:
This is also the proper place for the valueOf() or decode() function, instead of in the Command handler.